src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -20,48 +25,57 @@ const concatDescription = (position: SyPersonioJob): string => {
   );
 };
 
+/**
+ * Create and write structured data based on https://schema.org/JobPosting
+ *
+ * See the Google guide for additional insights:
+ * https://developers.google.com/search/docs/advanced/structured-data/job-posting#structured-data-type-definitions
+ */
 export const CareerDetailsStructuredData = ({
   position,
 }: CareerDetailsStructuredDataProps) => {
-  const employmentType = convertScheduleToEmploymentType(position.schedule);
-
-  const data = {
-    '@context': 'https://schema.org/',
+  const structuredData: WithContext<JobPosting> = {
```

Comment:
I included a typings package for the structured data which will help other folks to understand the properties.

Reply:
👍